README.md

```diff
@@ -34,7 +34,7 @@ Iceberg is under active development at the Apache Software Foundation.
 
 The core Java library that tracks table snapshots and metadata is complete, but still evolving. Current work is focused on adding row-level deletes and upserts, and integration work with new engines like Flink and Hive.
 
-The [Iceberg format specification][iceberg-spec] is being actively updated and is open for comment. Until the specification is complete and released, it carries no compatibility guarantees. The spec is currently evolving as the Java reference implementation changes.
+The [Iceberg format specification][iceberg-spec] is being actively updated (as the Java reference implementation changes) and is open for comments. Until the specification is complete and released, it carries no compatibility guarantees.
```

Comment:
Probably we can say there are compatibility guarantees for V1/V2 specs now, but itd be a bigger discussion how to phrase it.  So ok with this for now

Reply:
I think it was more clear before, with the second sentence as a separate note.

Reply:
"open for comment" is sounded good to me. I was actually just listening to a radio show about the evolution of words in English and this is one of the most common types. We apparently love to change our verbs into nouns and vice-versa.

Tldr; I think this one was fine as it was